debezium-core/src/main/java/io/debezium/pipeline/DataChangeEvent.java

```diff
@@ -5,22 +5,13 @@
  */
 package io.debezium.pipeline;
 
-import org.apache.kafka.connect.source.SourceRecord;
-
-public class DataChangeEvent {
-
-    private final SourceRecord record;
-
-    public DataChangeEvent(SourceRecord record) {
-        this.record = record;
-    }
-
-    public SourceRecord getRecord() {
-        return record;
-    }
+/**
+ * Intermediary representation of data change events, as exchanged between producer threads (e.g. reading the MySQL binlog) and consumer threads (e.g. the Kafka Connect polling loop).
+ *
+ * @author Gunnar Morling
+ *
+ */
+public interface DataChangeEvent {
```

Comment:
@gunnarmorling Over I like the solution but this interface seems to be a little bit anemic and I'd say the naming currently does not represent the trait that it provides.
How about making the interface generic and add method like
```
T getData()
```
?

Reply:
@jpechane, agreed in general; OTOH, there's no real advantage or need for adding that method right now. I'd say let's do this once actually needed.